lib/lotus/helpers/html_helper.rb

```diff
@@ -176,7 +185,7 @@ module Helpers
     #     #    </div>
     module HtmlHelper
       private
-      # Instantiate an HTML builder
+      # Instantiate an HTML builder / HTML Fragment
```

Comment:
This fragment name is an internal detail that we shouldn't document here.
For developers it's just a method that let them to build HTML.

Can you please remove it?